system/services/taskmanager/AdHocTaskManagerService.cfc

```diff
@@ -10,18 +10,21 @@ component displayName="Ad-hoc Task Manager Service" {
 
 // CONSTRUCTOR
 	/**
-	 * @siteService.inject siteService
-	 * @threadUtil.inject  threadUtil
-	 * @logger.inject      logbox:logger:taskmanager
- 	 * @executor.inject    presideAdhocTaskManagerExecutor
+	 * @siteService.inject   siteService
+	 * @sysConfigSrvc.inject systemConfigurationService
+	 * @threadUtil.inject    threadUtil
+	 * @logger.inject        logbox:logger:taskmanager
+ 	 * @executor.inject      presideAdhocTaskManagerExecutor
 	 */
 	public any function init(
 		  required any siteService
+		, required any sysConfigSrvc
```

Comment:
We can't use super class `$getPresideSetting()`, etc.? (I think we can and we don't need this service injected).